MainWindow.py

```
@@ -1,695 +1,695 @@
+			response:requests.Response = requests.get("https://api.github.com/repos/ginsakura/MUSYNCSave/releases/latest")
+			resJson = response.json()
+			if "tag_name" in resJson:
+				tergetVersion:str = response.json()["tag_name"].replace("rc",".")
+			elif "message" in resJson:
+				if resJson["message"][:23] == "API rate limit exceeded":
+					messageHead:str = "GitHub公共API访问速率已达上限";
+					message:str = "是否前往发布页查看是否存在更新？";
+					url:str = "https://github.com/Ginsakura/MUSYNCSave/releases/latest";
+					self.root.after(0, lambda _:(webbrowser.open(url) if messagebox.askyesno(messageHead, message) else _ ));
+					return;
+		except Exception as e:
```

Ruff (0.14.8)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)